iptables

```diff
@@ -20,6 +20,9 @@ iptables -A INPUT -i eth0 -p tcp --dport 902 -j REJECT --reject-with icmp-port-u
 #	icmp-admin-prohibited
 #	tcp-reset
 
+# To only allow ssh on the server and block all other ports
+iptables -A INPUT -p tcp --dport 22 -j ACCEPT 
```

Comment:
```suggestion
iptables --append <INPUT> --protocol tcp --dport 22 --jump <ACCEPT> 
```

Optional fix: missing `<` and `>` around placeholders.
Optional fix: missing long options.

Reply:
Thank you